### DISCUSSION TITLE
Can't authenticate

### DISCUSSION THREAD

**Discussion by Madbeefer** [alandtse/tesla#42] [Q&A]
I have a separate instance of home assistant I use to test new things out on, and I was able to get it to work there, but now when I go to install it on my non test environment I cannot get it to authenticate. I even added another account to my tesla account to get a different token and that wouldn't work. I've uninstalled and reinstalled countless times.. I did have the core tesla integration, which keeps asking me to reconfigure it.. Is there some files I should go delete or something??

**Comment by alandtse** [comment]
It's possible Tesla is blocking you because you're hitting it too much from that machine. Tesla is not friendly to unofficial apps. Uninstall the official integration and you may just have to wait a few days.
